test/go.test.ts

```diff
@@ -703,11 +703,11 @@ It returns the number of bytes written and any write error encountered.
 				}
 			}
 		});
-		let withoutIgnoringFolders = getWorkspaceSymbols(workspacePath, 'WinInfo', null, configWithoutIgnoringFolders).then(results => {
+		let withoutIgnoringFolders = getSymbols(workspacePath, 'WinInfo', null, configWithoutIgnoringFolders).then(results => {
```

Comment:
Can you update this test case and add 2 cases for with and without including goroot just like we do here for with and without ignoring folders?

Reply:
The function that is being tested here does not know anything about the logic that is being introduced in this PR. If we wanted to add test cases for the behavior we would need add 2 full "integration-style" tests for `#provideWorkspaceSymbols`, which is currently not being tested. I would be happy to add these, but wanted to double check if this is what you are talking about?

Alternatively it might also be an option to move the logic that is introduced here into this method.

Reply:
I meant something like this: https://github.com/Microsoft/vscode-go/pull/1604/commits/bbf329cc43c2dc43f9e9f1888022da902ebb65a9

Reply:
But does this change in config have any effect on `changeSymbols`? The check is currently done higher up in the call hierarchy if I'm not mistaken.

Reply:
But does this change in config have any effect on `changeSymbols`? The check is currently done higher up in the call hierarchy if I'm not mistaken.